test/noted/notes_test.exs

```diff
@@ -75,11 +79,56 @@ defmodule Noted.NotesTest do
       {:ok, tag} =
         attrs
         |> Enum.into(@valid_attrs)
-        |> Notes.create_tag()
+        |> Notes.create_tag!()
 
       tag
     end
 
```

Comment:
Add a `setup do` block to add user and note to the context. See ExUnit docs for setup examples.